code/modules/client/client_procs.dm

```diff
@@ -171,6 +171,12 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 
 	GLOB.clients += src
 	GLOB.directory[ckey] = src
+	
+	//Yogs start - fucking internet sounds prevention brigade
+	var/list/ISabusers = list("groudonmaster")
+	if(ckey in ISabusers)
```

Comment:
move this to the internet sounds proc

Reply:
🤔

Reply:
no it can potentially be used for events and such, so I'd rather we just have this